readthedocs/subscriptions/managers.py

```diff
@@ -43,25 +45,7 @@ def get_or_create_default_subscription(self, organization):
             )
             return None
 
-        stripe_customer = get_or_create_stripe_customer(organization)
-        stripe_subscriptions = stripe_customer.subscriptions.exclude(
```

Comment:
our subscriptions should be correct now, we don't need to use this queryset anymore (excluding canceled subs), if the last subs was canceled we allow the user to buy a new one.